lib/modules/platform/gerrit/client.ts

```diff
@@ -72,10 +72,13 @@ class GerritClient {
     return changes.body;
   }
 
-  async getChange(changeNumber: number): Promise<GerritChange> {
+  async getChange(
+    changeNumber: number,
+    extraOptions?: string[],
+  ): Promise<GerritChange> {
+    const options = [...this.requestDetails, ...(extraOptions ?? [])];
     const changes = await this.gerritHttp.getJson<GerritChange>(
-      `a/changes/${changeNumber}?` +
-        this.requestDetails.map((det) => `o=${det}`).join('&'),
+      `a/changes/${changeNumber}?` + options.map((det) => `o=${det}`).join('&'),
```

Comment:
use `getQueryString` for proper quoting

https://github.com/renovatebot/renovate/blob/2f49607fecb80757a1d2a3a5ee26dd0204c14cd3/lib/util/url.ts#L72